framework/src/main/java/com/github/dbmdz/flusswerk/framework/rabbitmq/MessageBroker.java

```diff
@@ -188,7 +193,8 @@ private void provideInputQueues() throws IOException {
   }
 
   private void provideOutputQueues() throws IOException {
-    for (String topic : routingConfig.getOutgoing().values()) {
+    for (String topic :
+        routingConfig.getOutgoing().values().stream().flatMap(List::stream).toList()) {
```

Comment:
Would a helper method like e.g. `Stream<String> RoutingProperties.allOutgoing()` be useful to make this easier to read?

Reply:
That's a very good suggestion, I've added the method. Thank you!